writer.go

```diff
@@ -64,6 +67,7 @@ func (w *Writer) WriteFrame(frame *lepton3.Frame) error {
 	return w.bldr.WriteFrame(fields, compFrame)
 }
 
+// Close closes the CPTV file
```

Comment:
Perhaps mention that Close must be called to ensure the file is flushed to disk.